README.md

```diff
@@ -396,6 +396,20 @@ Thread.sleep(5_000);
 thread.interrupt();
 ```
 
+## Java 8 Comletion API
+
+See whole examples [here](/src/test/java/examples/CompletionExample.java).
+
+```java
+ffmpeg.executeAsync().toCompletableFuture()
+        .thenAccept(res -> {
```

Comment:
Please, fix indentation.

Reply:
I see what you mean. The common practice in Java is to use 2 indents when continuing a statement on the next line, but I'll fix this to have the same indentation as other examples.